TRB_label_assist/regenerate_classification_performance_results.py

```diff
@@ -15,10 +16,11 @@
 labeled_trip_df_map = {}
 expanded_labeled_trip_df_map = {}
 expanded_all_trip_df_map = {}
+ct_entry={}
 for u in all_users:
     ts = esta.TimeSeries.get_time_series(u)
-    ct_df = ts.get_data_df("analysis/confirmed_trip")
-
+    ct_entry[u]=eamtr._get_training_data(u,None)    
+    ct_df = ts.to_data_df("analysis/confirmed_trip",itertools.chain(ct_entry[u]))    
```

Comment:
why do you need this change?

Reply:
This'll prevent two reads from the database. This is done in response to  https://github.com/e-mission/e-mission-eval-private-data/pull/37#discussion_r1312331029 . We need the data in df as well as in entry format.

Reply:
Removed the itertools part.